### ISSUE TITLE
package.json invalid "main" entry /lib/index.js'

### ISSUE THREAD

**Issue by nicolasigot** [cheeriojs/cheerio#1842]
Hello,

With the latest update there are some issues using cheerio
Cannot find module '/node_modules/cheerio/lib/index.js'. Please verify that the package.json has a valid "main" entry
Checking in the code and package.json the /lib/index.js file does not exist anymore.

Can you please review ?
Thanks!

**Comment by srana6** [comment]
We are seeing errors in our project
```
    Cannot find module 'cheerio' from 'node_modules/enzyme/build/Utils.js'

    Require stack:
      node_modules/enzyme/build/Utils.js
      node_modules/enzyme/build/ReactWrapper.js
      node_modules/enzyme/build/index.js
      jestsetup.js
```

**Comment by fb55** [comment]
There were two botched releases (sorry about that!), but `v1.0.0-rc.9` should make everything work as expected. Please try upgrading and let me know if the error persists.

**Comment by srana6** [comment]
Yes it worked with updating it to latest version

**Comment by fb55** [comment]
Awesome, thanks for confirming @srana6!

**Comment by nicolasigot** [comment]
It is working fine now. Thanks

**Comment by specious** [comment]
I'm running into this with 1.0.0-rc.10. It looks like it's not building automatically.  

Running `yarn && yarn build` in the cheerio directory makes it work.

**Comment by specious** [comment]
Downgrading to 1.0.0-rc.9 worked perfectly.

The directory looks very different. It looks like the development source tree was accidentally published to npm for 1.0.0-rc.10 instead of the deployment build.
